auth/gcloud/aio/auth/token.py

```diff
@@ -56,13 +58,16 @@ def get_service_data(service: Optional[str]) -> Dict[str, Any]:
             raise
 
         return {}
+    except TypeError:
+        data: Dict[str, Any] = json.loads(service.read())
```

Comment:
In case this fails (eg. the in-memory `service_file` is not json-loadable, we'll want to maintain the old behavior of returning and empty dictionary. Perhaps something like:
```suggestion
        try:
            data: Dict[str, Any] = json.loads(service.read())
            return data
        except Exception:  # pylint: disable=broad-except
            return {}
```
Alternatively, maybe we could nest this within the first `try` in order to avoid duplicating the final global `Exception` block -- open to anything that maintains the behavior!